### PR TITLE
Kafka upgrade docs refreshed

### PR DESCRIPTION
**Pull request by alenkacz**

### CODE REVIEW

File: repository/kafka/docs/v0.2/upgrade.md [hunk 1]
```diff
@@ -15,24 +15,7 @@ Upgrading can be done by patching the Kafka Instance that holds all the configur
...
-```
-Now there are two operator versions installed:
-```
-kubectl  get operatorversions.kudo.k8s.io
-
-NAME              AGE
-kafka-0.1.1       2d2h
-kafka-0.2.0       2m6s
-```
+The following upgrade procedure assumes one running Kafka cluster and one Kafka operator version already installed (version `kafka-0.1.1`). It also assumes one is running KUDO 0.5.0:
```
Comment (zmalik) [inline repository/kafka/docs/v0.2/upgrade.md]:
```suggestion
The following upgrade procedure assumes one running Kafka cluster and one Kafka operator version already installed (version `kafka-0.1.1`). It also assumes one is running KUDO 0.5.0 or later:
```

File: repository/kafka/docs/v0.2/upgrade.md [hunk 2]
```diff
@@ -56,9 +39,21 @@ Plan(s) for "kafka-fc6vzn" in namespace "default":
 
 To update the Kafka cluster from version `0.1.1` to `0.2.0`:
 
+Upgrade operator version from current to 0.2.0:
```
Comment (zmalik) [inline repository/kafka/docs/v0.2/upgrade.md]: can you remove the above line or merge into this one? as right now it is:
```
To update the Kafka cluster from version `0.1.1` to `0.2.0`:

Upgrade operator version from current to 0.2.0:
```

**Review by zmalik — CHANGES_REQUESTED** [review]
thank you @alenkacz ! 
LGTM just leaving two suggestions

**Comment by alenkacz** [comment]
@zmalik done :)

**Review by zmalik — APPROVED** [review]
LGTM! Thanks a lot @alenkacz

**Pull request merged**